programs/demos/timer/trunk/timer.asm

```diff
@@ -137,9 +137,9 @@ draw_window:
 
 ; DATA AREA
 
-if lang eq ru
-    title   db   '������',0
-else
+if lang eq ru_RU
+    title   db   '������',0
```

Comment:
![ru](https://github.com/KolibriOS/kolibrios/assets/61193648/d3379c08-0a70-472c-baef-fa32662ea196)
Was this title change intentional?

Reply:
Thanks. Missed that!